frontend/packages/ceph-storage-plugin/src/utils/kebab-actions.ts

```diff
@@ -0,0 +1,14 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { KebabAction } from '@console/internal/components/utils';
+import { PersistentVolumeClaimModel } from '@console/internal/models/index';
+import { ClonePVC } from './clone-workflow';
+
+export const getKebabActionsForKind = (resourceKind: K8sKind): KebabAction[] => {
+  const menuActions: KebabAction[] = [];
+  if (resourceKind) {
+    if (resourceKind && resourceKind.kind === PersistentVolumeClaimModel.kind) {
```

Comment:
```suggestion
    if (resourceKind?.kind === PersistentVolumeClaimModel?.kind) {
```

Reply:
`PersistentVolumeClaimModel.kind` will always be defined. So no need to check that.